tests/lib/rules/jsx-handler-names.js

```diff
@@ -69,6 +73,18 @@ ruleTester.run('jsx-handler-names', rule, {
   }, {
     code: '<TestComponent onChange={this.handlerChange} />',
     errors: [{message: 'Handler function for onChange prop key must begin with \'handle\''}]
+  }, {
+    code: '<TestComponent onChange={this.handle} />',
+    errors: [{message: 'Handler function for onChange prop key must begin with \'handle\''}]
+  }, {
+    code: '<TestComponent onChange={this.handle2} />',
+    errors: [{message: 'Handler function for onChange prop key must begin with \'handle\''}]
+  }, {
+    code: '<TestComponent onChange={this.handl3Change} />',
+    errors: [{message: 'Handler function for onChange prop key must begin with \'handle\''}]
+  }, {
+    code: '<TestComponent onChange={this.handle4change} />',
+    errors: [{message: 'Handler function for onChange prop key must begin with \'handle\''}]
```

Comment:
hm, we should probably change the error message to add " and be in camelCase", otherwise this one's pretty confusing

Reply:
I updated the error message to be more specific and removed the handleC test case that does pass, but should not be considered valid.

Reply:
if the test wrongly passes, we should move it to the "invalid" section, even if it's commented out for now.